src/__tests__/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { RawResult } from '../types'
+import * as clone from 'clone'
+import * as axe from 'axe-core'
+
+let _dummyData: RawResult[]
+export async function getDummyData(version = '3.1') {
+  if (!_dummyData) {
+    document.body.innerHTML = `
+      <h1>My page </h1>
+      <main>
+        <p>Some page</p>
+        <p><input type="text"> Failing input field</p>
+      </main>
+    `
+    axe.configure({
+      // @ts-ignore:
+      reporter: function(raw, _, callback) {
+        callback(JSON.parse(JSON.stringify(raw)))
+      }
+    })
+    // @ts-ignore:
+    _dummyData = await axe.run()
+  }
+  return clone(_dummyData).map((result: RawResult) => {
```

Comment:
Why `clone()` this?

Reply:
I'm only running axe once. cloning the data prevents any mutation from persisting between tests.